spyder/widgets/editor.py

```diff
@@ -965,11 +965,36 @@ def __modified_readonly_title(self, title, is_modified, is_readonly):
             title = "(%s)" % title
         return title
 
-    def get_tab_text(self, filename, is_modified=None, is_readonly=None):
+    def get_tab_text(self, index, is_modified=None, is_readonly=None):

```

Comment:
Hi @ccordoba12, CircleCi (pydocstyle) is giving me an error of docstring (D400), however the line that is pointing out (`968` in `widgets/editor.py`) can't finish in a period (is the signature of a method) . In this case what can I do to fix the code style issue?